content/sensu-go/5.21/operations/deploy-sensu/install-sensu.md

```diff
@@ -26,19 +26,31 @@ See [supported platforms][5] for more information.
 
 ## Architecture overview
 
+Sensu works differently from other monitoring and observability solutions.
+
+The **Sensu backend** is powered by an an embedded transport and [etcd][16] datastore.
+You do not need to provision each device, server, container, or sidecar you want to monitor or list each of them in the Sensu backend.
+Instead, you create [subscriptions][41] and define which checks to run for all infrastructure components that belong to each subscription.
+
+Then, you install **Sensu agents**: lightweight clients that run on the infrastructure components you want to monitor.
+Agents automatically register with Sensu as entities when you start them up and connect to the Sensu backend with no need for further provisioning.
+You only need to specify the IP address for the Sensu backend server and the subscriptions the agents belong to.
```

Comment:
With 6.0, you technically don't have to set `subscriptions` up front in agent.yml, you can add them to the agent's entity via the API later. Not sure if we want to "muddy the waters" here.